### DISCUSSION TITLE
choco install mecab?

### DISCUSSION THREAD

**Discussion by SpinozaWeb** [DrewThomasson/ebook2audiobook#15] [Q&A] [answered]
Hi!. I try install your project. But after "choco install mecab" I see "mecab not installed. The package was not found with the source(s) listed. Source(s): 'https://community.chocolatey.org/api/v2/'"
What am I doing wrong?

**Comment by DrewThomasson** [comment, marked as answer]
Oh my bad lol,

Idk thy I thought it would be on choco but I guess it's not 

Looks like your going to have to manually download and install it

[mecab-website](https://taku910.github.io/mecab/#download)

- And I'm not gona lie I don't even know if that'll make it work on windows.


## OR for a easier / better solution for windows just use docker

- just use the docker image, then you don't have to deal with any of these things, and you only have a single command install


### 🐳 Using Docker

You can also use Docker to run the eBook to Audiobook converter. This method ensures consistency across different environments and simplifies setup.

#### 🚀 Running the Docker Container

To run the Docker container and start the Gradio interface, use the following command:

 -Run with CPU only
```powershell
docker run -it --rm -p 7860:7860 --platform=linux/amd64 athomasson2/ebook2audiobookxtts:huggingface python app.py
```
 -Run with GPU Speedup (Nvida graphics cards only)
```powershell
docker run -it --rm --gpus all -p 7860:7860 --platform=linux/amd64 athomasson2/ebook2audiobookxtts:huggingface python app.py
```

This command will start the Gradio interface on port 7860.(localhost:7860)

#### 🖥️ Docker GUI 

<img width="1401" alt="Screenshot 2024-08-25 at 10 08 40 AM" src="https://github.com/user-attachments/assets/78cfd33e-cd46-41cc-8128-3820160a5e40">
<img width="1406" alt="Screenshot 2024-08-25 at 10 08 51 AM" src="https://github.com/user-attachments/assets/dbfad9f6-e6e5-4cad-b248-adb76c5434f3">


- hope that helps :)
